src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -494,6 +494,22 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
         GroupedCell: (props) =>
           TableGroupedCell({ ...props, outputType: 'Link' }),
       },
+      {
```

Comment:
The filtering on this table doesn't seem to work. When I add a minimum value of 1 it doesn't display anything. Not sure why though.

Reply:
Could you send a video/more info of the filtering not working? I've tried it myself and it seems to work fine. It does filter out null-ish values, so for columns with no data, no results would be displayed.

![Filter Table min 1 working](https://github.com/user-attachments/assets/7df63402-468e-4903-9e24-c71407b7fb5c)

Reply:
I am using the mock data found at `/catalogue/671a5890ac3e72a90dbfb737`. Which looks like

https://github.com/user-attachments/assets/3ac0c4ce-b802-4b56-b499-508b1178a50d

Reply:
I've found and fixed the issue regarding this bug, however I found another "bug" that also seems to exist on prod.
![image](https://github.com/user-attachments/assets/906c8a9d-1b09-40f9-983e-5268c4339f16)
For some reason setting negative numbers as upper bounds do not correctly filter out values. I don't know if this is a known issue; I've spent some time trying to resolve it, but the cause does not seem obvious.

Reply:
Good spot, I didn't know about that either 😅

Reply:
Any changes to get working etc, should be addressed in a separate PR